platooning_strategic_IHP/include/platoon_manager_ihp.h

```diff
@@ -245,6 +253,18 @@ namespace platoon_strategic_ihp
         */
         double getIHPDesPosFollower(double dt);
 
+        /**
+        \brief UCLA: Return joiner's desired position
```

Comment:
Need description of the parameter, incl units of measure.
Also, for return value, need more clarity on what the integer means.  It seems this would be the position # that it will cut in front of.  Are the numbers 1-indexed or 0-indexed?